blog/2023-11-13-introducing-stately-sky/index.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Introducing Stately Sky'
+description: Announcing Stately Sky, the new way to run your state machines in the cloud.
+authors: [gavin]
+tags: [new, sky, serverless, cloud, run, machines, state, stately]
+date: 2023-11-13
+slug: 2023-11-13-introducing-stately-sky
+image: /blog/2023-11-13-introducing-stately-sky.png
+---
+
+The Stately team is very excited to announce a new feature we’ve been working on for quite some time! Join us in welcoming Stately Sky to the Studio. Lovingly built with [PartyKit](https://www.partykit.io/), Sky is our new serverless platform for running workflows within the Studio. With Sky, users may now run their statecharts as live machines in minutes, complete with XState v5 actors and multiplayer support.
+
+<!--truncate-->
+
+## Why Sky: a seamless developer experience without the lock-in(Working title)
+
+### Developer Experience
```

Comment:
```suggestion
```

I’d take this heading out, partly because it’s saying something similar to the heading above, but also because assistive tech doesn’t love a heading under a heading (it likes some content in between).

Reply:
Excellent point! Removed!